lang/py/src/avro/schema.py

```diff
@@ -722,13 +812,27 @@ def make_avsc_object(json_data, names=None):
   if hasattr(json_data, 'get') and callable(json_data.get):
     type = json_data.get('type')
     other_props = get_other_props(json_data, SCHEMA_RESERVED_PROPS)
+    logical_type = None
+    if 'logicalType' in json_data:
+      logical_type = json_data.get('logicalType')
+      if logical_type != 'decimal':
+       raise SchemaParseException("Currently does not support %s logical type" % logical_type)
```

Comment:
Hi, are you hoping to support other Avro standard logical types soon ? I think this introduces a breaking change for any clients that were using Avro library for parsing records that contain logical types that are not "decimal". Previously I think Avro library was just returning the base type without failing, but with this check it will explicitly fail.

Happy to file a JIRA if needed.